src/display/editor/stamp.js

```diff
@@ -431,6 +431,42 @@ class StampEditor extends AnnotationEditor {
     const bitmap = this.#isSvg
       ? this.#bitmap
       : this.#scaleBitmap(width, height);
+
+    if (this._uiManager.hasMLManager && !this.hasAltText()) {
+      const offscreen = new OffscreenCanvas(width, height);
+      const ctx = offscreen.getContext("2d");
+      ctx.drawImage(
+        bitmap,
+        0,
+        0,
+        bitmap.width,
+        bitmap.height,
+        0,
+        0,
+        width,
+        height
+      );
+      offscreen.convertToBlob().then(blob => {
+        const fileReader = new FileReader();
+        fileReader.onload = () => {
+          const url = fileReader.result;
+          this._uiManager
+            .guess({
+              service: "image-to-text",
+              request: {
+                imageData: url,
+              },
+            })
+            .then(response => {
+              const altText = response?.output || "";
+              if (altText) {
+                this.altTextData = { altText, decorative: false };
+              }
```

Comment:
Given the async nature of this code, isn't it *at least theoretically* possible that the user has already manually modified the altText when we reach this code?
If so, wouldn't we be overwriting the user here which seems like a bad idea?

```suggestion
              if (altText && !this.hasAltText()) {
                this.altTextData = { altText, decorative: false };
              }
```